sites/libguides/libguides.css

```diff
@@ -37,6 +37,7 @@ body { padding-left: 0px; padding-right: 0px; }
 /* normalize width alignment of top parts to match the header and footer breakpoints */
 @media only screen and (min-width: 1500px){
   #s-lg-guide-header #s-lg-guide-header-info > h1,
+    #s-lg-guide-header #s-lib-public-header-title > h1,
     #s-lg-guide-header #s-lg-guide-header-info > div#s-lg-guide-desc-container {
```

Comment:
note: this is to address a libguides bug with search and profile pages: 

![image](https://user-images.githubusercontent.com/5565284/93618290-0eb6c300-f9a5-11ea-814b-41b803c35755.png)